packages/core/src/schema/models/documentVersions.ts

```diff
@@ -24,7 +24,7 @@ export const documentVersions = latitudeSchema.table(
     resolvedContent: text('resolved_content'),
     commitId: bigint('commit_id', { mode: 'number' })
       .notNull()
-      .references(() => commits.id),
+      .references(() => commits.id, { onDelete: 'cascade' }),
```

Comment:
⚠️ We were missing the cascade deletion of all the documents associated to a commit. We want this when you delete a draft commit